### PR TITLE
Add yaml file for Google App Engine deployment

### PR DESCRIPTION
**Pull request by karenfrilya97**

### CODE REVIEW

File: .gcloudignore [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# This file specifies files that are *not* uploaded to Google Cloud Platform
```
Comment (ifhuang) [inline .gcloudignore:1]: Is this file necessary to be checked in?

Comment (karenfrilya97) [reply .gcloudignore:1]: I'm not too sure actually. But I thought it'd be nice to check this in so anyone can deploy to AppEngine without making their own version of this file.

File: .gcloudignore [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+#
+.gcloudignore
+# If you would like to upload your .git directory, .gitignore file or files
+# from your .gitignore file, remove the corresponding line
+# below:
+.git
+.gitignore
+
+# Node.js dependencies:
+node_modules/
```
Comment (chshda) [inline .gcloudignore:17]: Might be better to add a new line at the end

Comment (karenfrilya97) [reply .gcloudignore]: Added! Thanks for catching this.
